CHANGELOG-3.0.md

```diff
@@ -0,0 +1,15 @@
+# CHANGELOG for 3.0.x
+
+This changelog references the relevant changes (bug and security fixes) done in 3.0 version.
+
+### 3.0.0 (2021-03-26)
+
+* This new version is only for PHP >=7.4.
+* Bump all dependencies
+* Replace `symfony/http-kernel` with `symfony/dependency-injection` and 
```

Comment:
A bundle not depending on `symfony/http-kernel` looks weird to me: the base bundle class is there

Reply:
Done